Test/git-issues/git-issue-1604.dfy

```diff
@@ -0,0 +1,130 @@
+// RUN: %dafny /compile:3 /rprint:"%t.rprint" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype Cell = Cell(x: int)
+
+predicate isEven(x: int) {
+  x % 2 == 0
+}
+// Ghost constraint
+type GhostEvenCell = c: Cell | isEven(c.x) witness Cell(0)
+
+// Compilable constraint
+type CompilableEvenCell = c: Cell | c.x % 2 == 0 witness Cell(0)
+
+predicate method ghostEvenCellIsOneOrMore(c: GhostEvenCell)
```

Comment:
Don't Dafny method names always start with an Uppercase? It seems like it when I look at the test suite.